src/Formatter/App/App.fsproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>qsfmt</AssemblyName>
+    <RootNamespace>QsFmt.App</RootNamespace>
```

Comment:
Does anyone have any preferences for what to call this? We could keep QsFmt and use the namespace `Microsoft.Quantum.QsFmt`. Or something like QsFormat or QsFormatter? Or a longer option is "Q# Formatter" with namespace `Microsoft.Quantum.QSharp.Formatter`...

Then there is also samarsha/qsfmt#2. :)

Reply:
We decided to keep QsFmt.